app/enterprise/2.1.x/plugins/oidc-okta.md

```diff
@@ -60,25 +71,40 @@ $ curl -i -X POST https://admin.kong.example/routes/ROUTE_ID/plugins --data name
   --data config.scopes="profile"
 ```
 
-Several pieces of configuration above must use values specific to your environment:
+Some of the configurations above must use values specific to your environment:
 
 * The `issuer` URL can be found from your Authorization Server settings.
-* `redirect_uri` should be the URI you specified earlier when configuring your app. You can view and edit this from the **General** page for your application.
-* For `client_id` and `client_secret` replace `YOUR_CLIENT_ID` and `YOUR_CLIENT_SECRET` with the client ID and secret shown in your Okta application's **General** page.
+* The `redirect_uri` should be the URI you specified earlier when configuring your app.
+You can view and edit this from the **General** page for your application.
+* For `client_id` and `client_secret`, replace `YOUR_CLIENT_ID` and `YOUR_CLIENT_SECRET`
+with the client ID and secret shown in your Okta application's **General** page.
 
-Visiting a URL matched by that route in a browser will now redirect to Okta's authentication site and return you to the redirect URI after authenticating.
+Visiting a URL matched by that route in a browser will now redirect to Okta's authentication
+site and return you to the redirect URI after authenticating.
 
-Additional plugin parameter to consider:
+Additional plugin parameters to consider:
 
-* The `auth_methods` parameter defines a lists of all the authentication methods that you want the plugin to accept. By default value is a list of all the supported methods. It is advisable to define this parameter with only the methods you wish to allow.
+* The `auth_methods` parameter defines a lists of all the authentication methods
+that you want the plugin to accept. By default, its value is a list of all the supported methods.
+It is advisable to define this parameter with only the methods you want to allow.
 
 ### Access Restrictions
 
-The configuration above allows users to authenticate and access the Route even though no consumer was created for them: any user with a valid account in the directory will have access to the Route. The OIDC plugin allows this as the simplest authentication option, but you may wish to restrict access further. There are several options for this:
+The configuration above allows users to authenticate and access the Route even though
+no Consumer was created for them: any user with a valid account in the directory
+will have access to the Route. The OIDC plugin allows this as the simplest authentication option,
+but you may wish to restrict access further. There are several options for this:
+
+- Domain Restrictions
+- Consumer Mapping
+- Pseudo-Consumer Mapping
```

Comment:
Recommend adding links to these sections. Not sure where to link to for domain restrictions though?

Reply:
> Recommend adding links to these sections. Not sure where to link to for domain restrictions though?

I think links aren't really necessary here because the content is right underneath it. I was just trying to have a better intro than a colon above headings 

![image](https://user-images.githubusercontent.com/3756245/94733103-de6c0e80-032c-11eb-86f0-eb383d3265fb.png)

I'll remove Domain. 

Many of these sections created by Aaron have very similar content that are candidates for includes. I had ported edits from the Azure section here but the domain does not apply in this Okta instance.